cmd/devp2p/internal/ethtest/eth66_suite.go

```diff
@@ -84,6 +86,10 @@ func (s *Suite) TestGetBlockHeaders_66(t *utesting.T) {
 func (s *Suite) TestSimultaneousRequests_66(t *utesting.T) {
 	// create two connections
 	conn1, conn2 := s.setupConnection66(t), s.setupConnection66(t)
+	defer func() {
+		conn1.Close()
+		conn2.Close()
+	}()
```

Comment:
`defer func() { ... }` is not necessary for this and creates two extra lines. You can use 

    defer conn1.Close()
    defer conn2.Close()

instead.